week-3/F-array-forEach/exercise.js

```diff
@@ -8,7 +8,19 @@
 */
 
 var arr = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
-
+function multiples(index) {
+  if (index % 3 === 0 && index % 5 === 0) {
+    console.log("FizzBuzz");
+  } else if (index % 5 === 0) {
+    console.log("Buzz");
+  } else if (index % 3 === 0) {
+    console.log("Fizz");
+  } else {
+    console.log(index);
+  }
```

Comment:
Great! Well done!

FizzBuzz is a famous way of testing your programming skills 😄